js/hubs/components/Hubs.js

```diff
@@ -84,7 +84,7 @@ const Hubs = () => {
                 align="left"
                 sx={{ padding: { md: "0 165px 40px", xs: "0px 0px 10px" } }}
               >
-                <Link href="/all">Hubs </Link>
+                <Link href="/all">Hubs</Link>
```

Comment:
<img width="562" alt="Screen Shot 2022-08-11 at 11 55 29 AM" src="https://user-images.githubusercontent.com/2960410/184176824-8cfe283c-ded4-42ed-a87b-13f3988ae3e6.png">

shouldn't remove space in copy in Hubs component

Reply:
my b